readme.md

```diff
@@ -65,33 +66,34 @@ __We strive to be informed by data as much as possible__
   * Whenever taking a decision try to gather some quantitative data. If quant is not available use qualitative data. If you fail at those two, only then use your guts and experience.
     
 
-__Collaborate!__
+__We are Deeply Collaborative!__
 
   * Product development is a team effort, we give each other feedback early and often, always respectfully
   * Example: If you touch a new area of the codebase, ask for help 
   * Example: When a feature is too complicated, pair
   * Example: Don’t get stuck for too long, ask for help.
 
   
-__Evolutionary Architecture__
+__We believe in Evolutionary Architectures__
   
   * We evolve our architecture and technology constantly as our business evolves. We experiment with new technologies at the edges of our systems and adopt carefully and for real business value.
   * Example: elm 
   * Example: single page css
   * Example: some architecture decisions 
     
 
-__Continuous Improvement__
+__We are deeply vested in Continuous Improvement__
   
   * We leave code better than we found it. We refactor before developing a feature, to ease its development. 
   
 
-__Reduce code and people dependencies__
+__We try to Reduce code and people dependencies__
   
   * We practice shared code and project ownership. Any team member should be able to work on any project interchangeably and we aim to rotate teams to ensure people have exposure to all parts of the codebase.
 
 
-
-__Healthy conflict__
+__We encourage Healthy conflicts__
   
   * We believe in a blameless culture; which includes holding each other accountable through respectful feedback
+  * We strongly believe in the principle of "Disagree and Commit". We are tenacious and make strong but respectful arguments. However, once a decision is determined, we commit to it completely in belief and execution
```

Comment:
I disagree with committing completely in belief. I think it’s fine to continue to disagree with a decision as long as it isn’t significantly impacting performance - and if it is then maybe the reasoning behind a decision is not clear enough to the individual. Having an extra discussion is more worth it than ‘expecting’ commitment.

I want to avoid this clause sounding a bit like “every team decision is final and you’re not allowed to argue it further” - at the same time I’m not disagreeing with the fact that compromises have to made.

Reply:
The movie "12 Angry Men" comes to my mind. In our case it's much easier to make a verdict, but there is still a possibility that the majority was wrong.

Reply:
I do think we have to make commitments that aren't always accepted by everyone, but perhaps we should alter this to handle changes in the future.

All disagreements should be noted for future reference and we will consider changing attitudes.

Reply:
Further to Christian's point, I think this principle should be phrased as something like:

> We strongly believe in the principle of "Productive Disagreement". We are tenacious and make strong but respectful arguments. However, debates should not drag on and get in the way of productivity. Firm decisions should be made as promptly as possible, and - while not everyone may agree with the decision - fully committed to by all until an appropriate occasion to reopen the debate arises.

Reply:
> For example: You argued hard to change a KPI against a goal for your team. But a collective decision was taken against your argument. You now own the decision wholly and commit to it with your team. However, ongoing concerns can still be raised in an appropriate, time-boxed manner such as via OfficeVibe or team retrospectives.

Reply:
Yeah I can support that viewpoint.

Reply:
I'd support Tom's phrasing on this. In its current form, this principle is the same as one of Amazon's principles (https://www.amazon.jobs/en-gb/principles). We should be able to define our own culture.

Reply:
@JadeDickinson  👍 You got me there :)  
I was trying to get the sentiment across almost from a product building and product decisions perspective. I really like the language around product disagreements